packages/headless/src/features/facets/range-facets/date-facet-set/interfaces/request.ts

```diff
@@ -1,10 +1,31 @@
-import {
-  RangeRequest,
-  BaseRangeFacetRequest,
-} from '../../generic/interfaces/request';
+import {BaseRangeFacetRequest} from '../../generic/interfaces/request';
 import {CurrentValues, Type} from '../../../facet-api/request';
+import {FacetValueState} from '../../../facet-api/value';
 
-export type DateRangeRequest = RangeRequest<string>;
+/**
+ * The options defining a value to display in a `DateFacet`.
+ */
+export interface DateRangeRequest {
+  /**
+   * The start value of the range.
+   */
+  start: string;
+
+  /**
+   * The end value of the range.
+   */
+  end: string;
```

Comment:
We should probably include the date format in those comments.

Reply:
Great suggestion, thanks 💯